doc/changelog/07-vernac-commands-and-options/13202-debug-infra.rst

```diff
@@ -0,0 +1,4 @@
+- **Added:**
+  :opt:`Debug` to control debug messages, functioning similarly to the warning system
```

Comment:
If some preexisting debug flags and options are replaced by this general mechanism, a complete list should be given in the changelog.